capstone-frontend/src/components/UserCard.jsx

```diff
@@ -7,36 +7,78 @@ import '../styles/Groups.css';
 class UserCard extends Component {
   removeMember = () => {
     const jsonBody = {
-      "remove_memberIDs": this.props.user.id, 
+      "remove_memberIDs": this.props.user.id,
       "id": this.props.club.id
     }
-    fetch(`/api/clubs`, {
-        method: "put", 
-        body: JSON.stringify(jsonBody)
-      })
+    fetch(`/api/clubs`, { method: "put", body: JSON.stringify(jsonBody) })
       .then(this.props.removeMember(this.props.user.id))
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
+  }
+
+  acceptMember = () => {
+    const jsonBody = {
+      "remove_requestIDs": this.props.user.id,
+      "add_memberIDs": this.props.user.id,
+      "id": this.props.club.id
+    };
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(jsonBody) })
+      .then(this.props.fetchData)
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
+  }
+
+  rejectMember = () => {
+    const jsonBody = {
+      "remove_requestIDs": this.props.user.id,
+      "id": this.props.club.id
+    };
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(jsonBody) })
+      .then(this.props.fetchData)
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
   }
 
   addFriend = () => {
     //TODO #86: Create friend functionality
   }
 
   render() {
-    const removeMember = this.props.club 
-                             && this.props.club.ownerID === window.localStorage.getItem("userID")
-                             && this.props.club.ownerID !== this.props.user.id
-                             && <Button id="remove-member" variant="danger" onClick={this.removeMember}>
-                                  Remove&nbsp;Member
-                                </Button>;
+    const isMember = this.props.club && this.props.club.memberIDs && this.props.club.memberIDs.includes(this.props.user.id);
+    const isRequester = this.props.club && !isMember && this.props.club.requestIDs && this.props.club.requestIDs.includes(this.props.user.id);
+    const removeMember = this.props.club
+      && this.props.club.ownerID === window.localStorage.getItem("userID")
+      && this.props.club.ownerID !== this.props.user.id
+      && <Button className="mt-2" cvariant="danger" onClick={this.removeMember}>
+        Remove&nbsp;Member
+         </Button>;
+    const requestButtons = this.props.club
+      && this.props.club.ownerID === window.localStorage.getItem("userID")
+      && this.props.club.ownerID !== this.props.user.id
+      && <div>
+        <Button className="mt-2" variant="primary" onClick={this.acceptMember}>
+          Accept&nbsp;Member
+           </Button>
+        <Button className="mt-2" variant="danger" onClick={this.rejectMember}>
+          Reject&nbsp;Member
+           </Button>
```

Comment:
nit: Your opening button statement doesn't line up with your closing button statement, use Gitpod's format document

Reply:
Seems like format document did this, which is strange. I'm pretty sure when I wrote them they were lined up, and then format document changed it to this. Since now when I call format nothing changes. I'll fix it!